BUILD/monsters/sniff.dat

```diff
@@ -18,3 +18,4 @@ War Hippy Naturopathic Homeopath	path:Kingdom of Exploathing;item:filthy poultic
 War Hippy Homeopath	path:Kingdom of Exploathing;item:filthy poultice<5
 Possessed Wine Rack
 Blue Oyster cultist
+Dirty Old Lihc	prop:cyrptNicheEvilness>25
```

Comment:
nit: I think >25 is too loose here: it's not worth olfacting a Dirty Old Lihc even if we're at, say, 28 evilness remaining - since killing the current Lihc will get us to 25

Reply:
heh good point. That's what happens when I write code on autopilot.